### ISSUE TITLE
Activation via wp-cli breaks embed code

### ISSUE THREAD

**Issue by jcmb42** [epiphyt/embed-privacy#53]
Just noticed: Activation of the plugin via `wp plugin activate embed-privacy` without specifying `--user` leads to an error message (propably by [capability verification](https://github.com/epiphyt/embed-privacy/blob/0d04849708354aa9d6f83a7a006390f0c8e863ff/inc/class-fields.php#L416)):

`Error: You are not allowed to edit an embed.`

However the plugin still seems to get activated but remains in a broken state until uninstalled (missing code in generated html).

[Here's](https://pastebin.com/7tjs5Qax) a diff between broken and working embed code.

EDIT (screenshots attached):

plugin active, but broken
<img width="1311" alt="Bildschirmfoto 2021-03-24 um 14 28 43" src="https://user-images.githubusercontent.com/7958968/112318582-78de5680-8cad-11eb-8140-4cf5cfb7068c.png">

plugin working
<img width="1440" alt="Bildschirmfoto 2021-03-24 um 14 26 23" src="https://user-images.githubusercontent.com/7958968/112318573-77ad2980-8cad-11eb-8d71-3706cd6ba1df.png">

**Comment by MatzeKitt** [comment]
Thank you for reporting this issue!

In order to get it working, please uninstall the plugin first if you didn’t do yet since the plugin data in the database is incomplete. Then, reinstall and activate it manually via admin or use the upcoming version 1.3.1, which provides a fix for it.

**Comment by jcmb42** [comment]
Thank you for looking into this so quickly. I'm afraid the fix didn't solve the bug though... 

I deactivated & uninstalled the plugin, installed version 1.3.1 and verified the existence of the corresponding [code merge](https://github.com/epiphyt/embed-privacy/pull/59/commits/5c95b026150b603182cf5d8a992cb119aef7cc9f) on the server in inc/class-fields.php. I then activated the plugin and the message still appears (plus embeds get scrambled).


Not specifying a user when activating via command-line means the condition is still met, even though WP-CLI is defined.
I could just add `--user=admin` or any superadmin ID but as far as I know that shouldn't be necessary.

I fiddled around with `! current_user_can( 'edit_posts', $post_id ) && ! (defined( 'WP_CLI' ) && WP_CLI)` (i.e. capability check should apply on plugin activation via webinterface only).

This seems to work for me but neither am I a plugin developer, nor do I know the consequences for the whole plugin structure - especially security-wise...

Any hints?
